src/einsteinpy/coordinates/core.py

```diff
@@ -30,13 +30,22 @@ def __init__(self, x, y, z):
         self.z = z
         super().__init__(x.si.value, y.si.value, z.si.value)
         self.system = "Cartesian"
+        self.coordinates = {
```

Comment:
I guess the self.coordinates is a vague way of indicating what's already coordinate!

Reply:
Should I go with `self.data` ?